<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ANKUSH-PC" failures="1" tests="1" name="SeleniumClass9" time="0.047" errors="0" timestamp="27 Oct 2018 08:01:22 GMT">
  <testcase classname="com.testng.db.SeleniumClass9" name="Samplemethod" time="0.015">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.testng.db.SeleniumClass9.Samplemethod(SeleniumClass9.java:12)
at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.base/java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- Samplemethod -->
</testsuite> <!-- SeleniumClass9 -->
